composer.json

```diff
@@ -24,16 +24,17 @@
         }
     ],
     "require-dev": {
-        "doctrine/coding-standard": "^5.0.1",
-        "infection/infection": "^0.9.3",
+        "doctrine/coding-standard": "^6.0.0",
+        "infection/infection": "^0.10.6",
```

Comment:
Note: newer `infection/infection` versions fail with bizarre issues - didn't go there (for now)

Reply:
What bizarre issues does it fail with? i'd like to fix those if possible

Reply:
@BackEndTea can you run https://github.com/Roave/BackwardCompatibilityCheck/pull/122/commits/0138f44bc17a235637e5c80511032061af349fdb `vendor/bin/infection`? It fails with a fatal error on a dynamic call from an extracted AST node.

The error message I got is in https://github.com/Roave/BackwardCompatibilityCheck/pull/122/commits/93788f8d5aaff5e8ceaf205f9d02a68bcc4e055b

Reply:
Ah, that looks like an error with anonymous/internal classes. Should be fixed in 0.13

Reply:
@Ocramius With the release of 0.13, that version can be used now.

Reply:
Thanks @BackEndTea!